infrastructure/variables.tf

```diff
@@ -85,6 +85,16 @@ variable "environment" {
   type        = string
 }
 
+variable "front_door_config" {
+  description = "Config for the frontdoor in tooling subscription"
+  type = object({
+    subscription_id   = string
+    frontdoor_name    = string
+    frontdoor_rg      = string
+    frontdoor_ep_name = string
```

Comment:
Minor thing, but if the variable is called "front_door_config" I don't think we need to repeat it, how about:

```
variable "front_door_config" {
  description = "Config for the frontdoor in tooling subscription"
  type = object({
    name = string
    rg      = string
    endpoint_name = string
    })
  }
  ```